index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                <ul style="list-style: none;" class="language-button">
+                    <li><button type="submit" class="html-button">html</button></li> 
+                <li><button type="submit" class="css-button">css</button></li>
+              <li><button type="submit" class="javascript-button">javascript</button></li>
+            </ul>  
+            <button type="submit" class="project-button">See Project</button>
+        </div>
+</div>        
+<div class="grid-container3"> 
+            <div class="grid-item1"><img src="Snapshoot Portfolio(3).png" alt="portfolio"></div>
+            <div class="grid-item2">
+        <h2>Tonic</h2>
+            <ul >
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <li><button type="submit" class="html-button">html</button></li> 
+                <li><button type="submit" class="css-button">css</button></li>
+              <li><button type="submit" class="javascript-button">javascript</button></li>
+            </ul>   
+            <button type="submit" class="project-button">See Project</button>
+         </div>
+</div>         
+<div class="grid-container4"> 
+    <div class="grid-item1"><img src="Snapshoot Portfolio(4).png" alt="portfolio"></div>
+          <div class="grid-item2"><h2>Multi-Post <br>Stories</h2>
+     <ul>
+         <li style="list-style: none;" id="canopy">CANOPY</li>
+         <li>Back End Dev</li>
+         <li>2015</li>
+     </ul>
+     <p>A daily selection of privately<br>
+          personalized reads; no accounts or<br>
+           sign-ups required.</p>
+           <ul style="list-style: none;" class="language-button">
+             <li><button type="submit" class="html-button">html</button></li> 
+         <li><button type="submit" class="css-button">css</button></li>
+       <li><button type="submit" class="javascript-button">javascript</button></li>
+     </ul>  
+     <button type="submit" class="project-button">See Project</button> 
+   </div>
+</div>
+</section>
+<section class="section-3">
+        <h1>About <br>Myself</h1>
+    <p>I’m a software developer! I can help you <br>
+        build a product , feature or website Look <br>
+        through some of my work and <br>
+        experience! If you like what you see and <br>
+        have a project you need coded, don’t <br>hestiate to contact me.</p>
```

Comment:
[OPTIONAL]
- [ ] This is marked as optional since the previous reviewer didn't notice this, it's better if you add `font-family` and `color` to this paragraph to match the one on Figma 👍 

## Your design;
![image](https://user-images.githubusercontent.com/50721479/193118331-bc653e18-9b01-4a32-b463-1002489fc3e1.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/193118387-806da1e0-0990-4ea8-8e8d-1cde86f022d1.png)